ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,216 @@
+                IAA #: AOD 16037; Federally Qualified Health Centers: HHSN 263201600085U; Data and Research Center:
+                5 U2C OD023196; Biobank: 1 U24 OD023121; The Participant Center: U24 OD023176; Participant Technology
+                Systems Center: 1 U24 OD023163; Communications and Engagement: 3 OT2 OD023205; 3 OT2 OD023206;
+                and Community Partners: 1 OT2 OD025277; 3 OT2 OD025315; 1 OT2 OD025337; 1 OT2 OD025276. In addition,
+                the <AoUTitle/> would not be possible without the partnership of its participants.”
+              </li>
+              <li>
+                I will submit an electronic version of a final, peer-reviewed manuscript to PubMed Central
+                immediately upon acceptance for publication, to be made publicly available no later than 12
+                months after the official date of publication.
+              </li>
+            </IndentedUnorderedList>
+          </li>
+        </IndentedUnorderedList>
+      </li>
+    </IndentedUnorderedList>
+    <SecondHeader>As “Authorized Demonstration User” of the <AoUTitle/> data, I
+      will:</SecondHeader>
+    <IndentedUnorderedList>
+      <li><strong>NOT</strong> share my login information with anyone.
+        <IndentedUnorderedList>
+          <li>I will not share my login information with another authorized demonstration or other user
+            of the <AoUTitle/>.
+          </li>
+          <li>I will not create any group or shared accounts.</li>
+        </IndentedUnorderedList>
+      </li>
+      <li>
+        <strong>NOT</strong> use <AoUTitle/> data or any external data, files, or software that I upload
+        into the Research Workbench for research that is discriminatory or stigmatizing of individuals, families,
+        groups, or communities. Please review the All of Us policy on stigmatizing research here.
+        <IndentedUnorderedList>
+          <li>I will contact the <AoUTitle/> Resource Access Board (RAB) for
+            further guidance on this point as needed.
+          </li>
+        </IndentedUnorderedList>
+      </li>
+      <li><strong>NOT</strong> attempt to contact <AoUTitle/> participants.
+      </li>
+      <li><strong>NOT</strong> make copies of or download participant-level data and remove it from
+        the <AoUTitle/> environment.
+        <IndentedUnorderedList>
+          <li>I will not take screenshots or attempt any other way of
+            copying participant-level data.
+          </li>
+        </IndentedUnorderedList>
+      </li>
+      <li><strong>NOT</strong> redistribute or publish registered or controlled tier <AoUTitle/> data including
+        aggregate statistics that are more granular than buckets of 20 individuals without explicit approval from
+        the <AoUTitle/>.
+      </li>
+      <li><strong>NOT</strong> sell or distribute <AoUTitle/> data at any level of granularity for the purpose
```

Comment:
not sure if needed but all of the "NOT"'s under the `As “Authorized Demonstration User” of the All of Us Research Program data, I will:
` section has an underline

Reply:
They don't. Jay checked with Dikshya on that